client/modules/IDE/components/FileNode.jsx

```diff
@@ -256,18 +274,15 @@ class FileNode extends React.Component {
             { isFolder &&
               <div className="sidebar__file-item--folder">
                 <button
-                  className="sidebar__file-item-closed"
-                  onClick={this.showFolderChildren}
-                  aria-label={t('FileNode.OpenFolderARIA')}
-                >
-                  <FolderRightIcon className="folder-right" focusable="false" aria-hidden="true" />
-                </button>
-                <button
-                  className="sidebar__file-item-open"
-                  onClick={this.hideFolderChildren}
-                  aria-label={t('FileNode.CloseFolderARIA')}
+                  className={folderButtonClass}
```

Comment:
I think maybe this should have `focusable=false`? I don't like that when you tab through the sidebar, you can focus on the folder name, it feels superfluous for keyboard users when you can just click on the arrow to the left!